backend/app.py

```diff
@@ -0,0 +1,69 @@
+from flask import Flask, request
+from flask_restful import Resource, Api
+
+app = Flask(__name__)
+api = Api(app)
+
+def Response(result, error):
+    return { 'result': result, 'error': error }
+
+class SignIn(Resource):
+    def post(self):
+        email = request.form['email']
+        password = request.form['password']
+        return Response(None, '{"Not implemented"}')
+
+
+class SignUp(Resource):
+    def post(self):
+        email = request.form['email']
+        password = request.form['password']
+        password2 = request.form['password2']
+        return Response(None, '{"Not implemented"}')
+
+
+class Posts(Resource):
+    def get(self):
+        limit = request.form['limit']
+        offset = request.form['offset']
+
+        return Response(None, '{"Not implemented"}')
+    
+    def post(self):
+        xtoken = request.headers['xtoken']
+
+        title = request.form['title']
+        description = request.form['description']
+        body = request.form['body']
+        image = request.form['image']
+        
+        return Response(None, "{'Not implemented'}")
+
+
+class Post(Resource):
```

Comment:
@khohryakov please move all api controller classes inside `api/controllers` folder

we will implement MVC pattern using SQLAlchemy for models